manifest.json

```diff
@@ -32,7 +32,19 @@
     },
     "projectOptions": ["app", "addon"],
     "nodeVersion": "4.0.0",
-    "commands": ["github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"]
+    "commands": [
+      " github:rondale-sc/es5-getter-ember-codemod es5-getter-ember-codemod app/**/*.js"
+    ]
+  },
+  "notify-property-change": {
+    "versions": {
+      "ember-source": "3.1.0-beta.1"
+    },
+    "projectOptions": ["app", "addon"],
+    "nodeVersion": "4.0.0",
```

Comment:
Are you sure ember-3x-codemods works in node 4? I don't see anything in that project that says which node versions are supported.

Reply:
Since it is using codemod-cli  0.2.10, which requires node 6, will update the same
https://github.com/rwjblue/codemod-cli/blob/v0.2.10/package.json